src/main/java/com/google/growpod/servlets/ResetDataServlet.java

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: 2020 throughout

Reply:
I forgot to make changes to my new code based on your comments on #26. This applies to many of your comments here.